4. User API/userAPI/src/main/kotlin/UserRepository.kt

```diff
@@ -0,0 +1,27 @@
+class UserRepository {
```

Comment:
every method should have at-most 80 lines of code in it, if it is exceeding then create another method.
Every class must have at-most 500 lines of code, else make another class/service.
download sonarlint and install it in your ide to get indications to write cleaner and readable code.